Create an inspector rule to set the expected BIOS version

dougszumski opened the pull request:
Only PowerEdge R630s are present in the alt-1 system. The conditions
will need updating if this is cherry-picked to production.

dougszumski committed:
Create an inspector rule to set the expected BIOS version

Only PowerEdge R630s are present in the alt-1 system. The conditions
will need updating if this is cherry-picked to production.

markgoddard commented on etc/kayobe/inspector.yml:
```diff
@@ -100,6 +100,16 @@ inspector_lldp_switch_port_interface_map:
+    conditions:
+    - field: "data://inventory.system_vendor.product_name"
+      op: "matches"
+      value: "^PowerEdge R630$"
```
Can we make this the same in alt-1 and production? Shouldn't have any negative consequences and reduces the diff and potential merge issues.

As it stands, all nodes should have the same BIOS version shouldn't they?

dougszumski replied:
Sounds good - I think it's the same for the R630/730 series, but doesn't that C series node have a different BIOS? Shouldn't be a problem either way.

markgoddard replied:
C4130 was a different binary but was also 2.6.0

dougszumski replied:
Thanks, I gave it it's own rule just in case it diverges in the future.

dougszumski committed:
Add support for production nodes

The C4130 has the same BIOS version as the R series nodes, but in
the future it may diverge hence why it has it's own rule.
